[NET45PLUS] Antlr4.Runtime.Atn.ATNState has no definition for Transitions

Got the build error when building with following msbuild properties:

   TargetFrameworkVersion=v4.5;DefineConstants="TRACE;NET35PLUS;NET40PLUS;NET45PLUS"

Misc\RuleDependencyChecker.cs(380,57): error CS1061: 'Antlr4.Runtime.Atn.ATNState' enthält keine Definition für 'Transitions', und es konnte keine Erweiterungsmethode 'Transitions' gefunden werden, die ein erstes Argument vom Typ 'Antlr4.Runtime.Atn.ATNState' akzeptiert (Fehlt eine Using-Direktive oder ein Assemblyverweis?). [antlr4-csharp\Runtime\CSharp\Antlr4.Runtime\Antlr4.Runtime.vs2013.csproj]

TransitionsArray has to be used.

PR in preparation.

Regards,
ViceIce

--- a/runtime/CSharp/Antlr4.Runtime/Misc/RuleDependencyChecker.cs
+++ b/runtime/CSharp/Antlr4.Runtime/Misc/RuleDependencyChecker.cs
@@ -377,7 +377,7 @@ namespace Antlr4.Runtime.Misc
                 {
                     continue;
                 }
-                foreach (Transition transition in state.Transitions)
+                foreach (Transition transition in state.transitions)
                 {
                     if (transition.TransitionType != TransitionType.Rule)
                     {
@@ -932,7 +932,7 @@ namespace Antlr4.Runtime.Misc
                 {
                     continue;
                 }
-                foreach (Transition transition in state.TransitionsArray)
+                foreach (Transition transition in state.transitions)
                 {
                     if (transition.TransitionType != TransitionType.Rule)
                     {

